integration-tests/src/test/java/org/openmrs/module/fhir2/providers/r4/EncounterFhirResourceProviderIntegrationTest.java

```diff
@@ -47,8 +47,7 @@
 
 public class EncounterFhirResourceProviderIntegrationTest extends BaseFhirR4IntegrationTest<EncounterFhirResourceProvider, Encounter> {
 	
-	private static final String[] ENCOUNTER_DATA_XML = {
```

Comment:
This probably required a longer explanation, I'll add it below on the test data commit.